src/transport_performance/utils/defence.py

```diff
@@ -120,7 +126,12 @@ def _is_expected_filetype(pth, param_nm, check_existing=True, exp_ext=".zip"):
     _, ext = os.path.splitext(pth)
     if check_existing and not os.path.exists(pth):
         raise FileExistsError(f"{pth} not found on file.")
```

Comment:
This isn't being tested with `Test_IsExpectedFiletype`. This would be a useful test to have at the lowest level.

Reply:
Good spot! Will check asap